S/StringDistances/Compat.toml

```diff
@@ -11,8 +11,11 @@ julia = "0.4.1-0.7"
 ["0.2"]
 julia = "0.6-0.7"
 
-["0.2-0.3"]
+["0.2-0.3.1"]
 IterTools = "0.1-1.1"
 
-["0.3"]
+["0.3.0-0.3.1"]
 julia = "0.7-1.3"
+
+["0.3.2"]
+julia = "0.7-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.